crashtracker/src/configuration.rs

```diff
@@ -18,11 +18,35 @@ pub struct CrashtrackerConfiguration {
     pub collect_stacktrace: bool,
     pub create_alt_stack: bool,
     pub endpoint: Option<Endpoint>,
-    pub path_to_receiver_binary: String,
     pub resolve_frames: CrashtrackerResolveFrames,
+    pub timeout: Duration,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct CrashtrackerReceiverConfig {
```

Comment:
As mostly an end-user, this makes sense.

Reply:
👍